content/docs/guides/scim/okta.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Configuring SCIM in Okta
+meta_desc: This page describes how to support SCIM 2.0 functionality between Pulumi and Okta.
+menu:
+    userguides:
+        parent: scim
+        weight: 1
+---
+
+This document covers steps to help you configure automatic provisioning/deprovisioning of your users and groups in Pulumi using SCIM 2.0.
+
+Please note that not all SCIM features are supported. For more information, see [Known Limitations]({{< relref "#known-limitations" >}}).
+
+## Prerequisites
+
+* Your organization must already be configured to use [SAML SSO]({{< relref "/docs/guides/saml/okta" >}}) with Pulumi.
+* You must be an admin of your organization.
+* (Optional) It is highly recommended that you have more than one admin for your org.
+
+## Enabling SCIM For Your Okta Org
+
+In order to configure SCIM for your Okta organization, you may need to email support@okta.com and request that they enable `SCIM_PROVISIONING` for the organization.
+
+Once the feature is enabled, navigate to the Pulumi SAML app configured in your Okta org from the **Classic UI**. Under the **General** tab, in the **App Settings** dialog box, locate the **Provisioning** section and select **SCIM**. Click **Save** to save your changes and close the **App Settings** dialog box.
+
+For more information on how to enable SCIM provisioning in Okta, see [How to Enable SCIM Provisioning in Custom App](https://support.okta.com/help/s/article/SCIM-Provisioning-Enabled-in-Custom-App) in the Okta documentation.
+
+## Changing the SAML SSO Settings For SCIM
+
+As part of the setup process, you need to adjust some of the parameters sent by a SAML SSO request. To make the changes, navigate to the **SAML Settings** section, click **Edit**, and make the following changes:
+
+1. Click **Next** on the first step.
+2. On Step 2, in the **General** section, set the application username to **Okta username prefix**.
+3. In the **Attribute Statements** section, click **Add Another** and set the **Name** of the property to `email` and the value to be source from `user.email`. The ATTRIBUTE STATEMENTS should now be as follows:
+
+    | Name | Name Format | Value |
+    | --- | --- | --- |
+    | firstName | Basic | user.firstName |
+    | lastName | Basic | user.lastName |
+    | email | Basic | user.email |
+
+4. Click **Next** then click **Finish**.
+
+## Configuring the SCIM Connector
+
+To configure the SCIM connector, click the **Provisioning** tab, and then select the **Integration** option in the left nav. Click **Edit**. Fill out the form as follows:
+
+* SCIM connector base URL: `https://api.pulumi.com/scim/v2/{orgName}`, where `{orgName}` must be replaced with your organization’s login name (not display name). If you do not know this, navigate to your SAML settings and look at the SSO URL. It will have your organization’s login name in the URL.
+* Supported provisioning actions: check **Push New Users** and **Push Profile Updates**.
```

Comment:
Should we also tell users to enable "Push Groups" here if they are planning to use that feature as well rather than having to come back here later anyway?

Reply:
@praneetloke I think it's best to provide a link to the group provisioning section here given the extra look-up steps that seem to be involved.